boxsdk/auth/oauth2.py

```diff
@@ -14,6 +14,17 @@
 from boxsdk.network.default_network import DefaultNetwork
 from boxsdk.config import API
 from boxsdk.exception import BoxOAuthException
+from boxsdk.util.text_enum import TextEnum
+
+
+class TokenScope(TextEnum):
```

Comment:
Docstring, and link to documentation that defines these.

Reply:
done